solution/zenkoversion.yaml

```diff
@@ -132,7 +132,6 @@ spec:
       exporter:
         image: redis_exporter
         tag: '${REDIS_EXPORTER_TAG}'
-      kubedb: '${REDIS_KUBEDB_TAG}'
```

Comment:
must also be done in the other zenkoversion.yaml

Reply:
What do you mean by "the other zenkoversion.yaml" ?

Reply:
there are 2 files named `zenkoversion.yaml` in the repo : the other one must be updated as well